code/modules/mob/living/basic/space_fauna/spider/giant_spider/giant_spiders.dm

```diff
@@ -246,6 +247,97 @@
+	icon_state = "tank"
+	icon_living = "tank"
+	icon_dead = "tank_dead"
+	maxHealth = 500
+	health = 500
+	damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 1, CLONE = 1, STAMINA = 1, OXY = 1)
+	melee_damage_lower = 5
+	melee_damage_upper = 5
+	obj_damage = 15
+	speed = 5
+	player_speed_modifier = -4
+	menu_description = "Extremly tanky with very little damage able to self heal and lay reflective silk screens."
+
+/mob/living/basic/spider/giant/tank/Initialize(mapload)
+	. = ..()
+	var/datum/action/cooldown/mob_cooldown/lay_web/web_reflector/reflector_web = new(src)
+	reflector_web.Grant(src)
+
+	var/datum/action/cooldown/mob_cooldown/lay_web/web_passage/passage_web = new(src)
+	passage_web.Grant(src)
+
+	AddElement(/datum/element/web_walker, /datum/movespeed_modifier/below_average_web)
+
+	AddComponent(/datum/component/healing_touch,\
+		heal_brute = 50,\
+		heal_burn = 50,\
+		heal_time = 5 SECONDS,\
+		self_targetting = HEALING_TOUCH_SELF_ONLY,\
+		interaction_key = DOAFTER_SOURCE_SPIDER,\
+		valid_targets_typecache = typecacheof(list(/mob/living/basic/spider/growing/young/tank, /mob/living/basic/spider/giant/tank)),\
+		extra_checks = CALLBACK(src, PROC_REF(can_mend)),\
+		action_text = "%SOURCE% begins mending themselves...",\
+		complete_text = "%SOURCE%'s wounds mend together.",\
+	)
+/// Prevent you from healing other tangle spiders, or healing when on fire
+/mob/living/basic/spider/giant/tank/proc/can_mend(mob/living/source, mob/living/target)
+	if (on_fire)
+		balloon_alert(src, "on fire!")
+		return FALSE
+	return TRUE
+
+/**
+ * ### Spider Breacher
+ * A subtype of the giant spider which is similar on every single way,
+ * This spider is only slightly slower than a human.
+ */
+/mob/living/basic/spider/giant/breacher
+	name = "breacher spider"
+	desc = "Furry and light brown with a white dark brown highlights, it makes you shudder to look at it. This one has bright red eyes."
+	icon_state = "breacher"
+	icon_living = "breacher"
+	icon_dead = "breacher_dead"
+	maxHealth = 120
+	health = 120
+	melee_damage_lower = 5
+	melee_damage_upper = 10
+	unsuitable_atmos_damage = 0
+	minimum_survivable_temperature = 0
+	maximum_survivable_temperature = 700
+	unsuitable_cold_damage = 0
+	wound_bonus = 25
+	bare_wound_bonus = 50
+	sharpness = SHARP_EDGED
+	obj_damage = 60
+	web_speed = 0.25
+	limb_destroyer = 50
+	speed = 5
+	player_speed_modifier = -4
+	sight = SEE_TURFS
+	menu_description = "Atmos resistant with the ability to destroy walls and limbs alike, with the ability to send warnings to the nest."
+	var/datum/action/cooldown/mob_cooldown/charge/basic_charge/charge
+
+/mob/living/basic/spider/giant/breacher/Initialize(mapload)
+	. = ..()
+	var/datum/action/cooldown/mob_cooldown/lay_web/solid_web/web_solid = new(src)
+	web_solid.Grant(src)
+
+	var/datum/action/cooldown/mob_cooldown/command_spiders/warning_spiders/spiders_warning = new(src)
+	spiders_warning.Grant(src)
+
+	AddElement(/datum/element/tear_wall)
```

Comment:
> Wait do we have `/datum/element/tear_wall` AND `/datum/element/wall_tearer`? Fuck `wall_tearer` is cooler though.

whats the difference?

I just used the one tarantula had since it was already there for spiders

Reply:
`wall_tearer` has animations and `tear_wall` doesn't, there's also now a PR open removing `tear_wall` for being redundant so I would change it 😅